assets/js/modules/analytics/setup.js

```diff
@@ -796,12 +796,8 @@ class AnalyticsSetup extends Component {
 			onSettingsPage,
 			isEditing,
 		} = this.props;
-		const disabledProfile = ! isEditing;
-
-		let disabledProperty = ! isEditing;
-		if ( existingTag && selectedProperty ) {
-			disabledProperty = true;
-		}
+		const disabledProperty = ! isEditing || parseInt( selectedAccount, 10 ) < 1 || ! selectedAccount || ( existingTag && selectedProperty );
+		const disabledProfile = ! isEditing || ! selectedProperty;
```

Comment:
```suggestion
		const disabledProfile = ! isEditing || ~~selectedProperty < 1;
```

Reply:
@aaemnnosttv How about `disabledProfile = ! isEditing || ! selectedProperty || disabledProperty;`?

Reply:
We want to test for `0` or `-1` as well; they're magic constant values used to set up a new property. If that's enabled we shouldn't allow the user to select a view.